ksql-rest-model/src/test/java/io/confluent/ksql/rest/entity/ServerClusterIdTest.java

```diff
@@ -29,14 +31,17 @@ public void shouldReturnServerClusterId() {
 
     // When:
     final String id = serverClusterId.getId();
-    final Map<String, String> scope = serverClusterId.getScope();
+    final Map<String, Object> scope = serverClusterId.getScope();
 
     // Then:
     assertEquals("", id);
     assertEquals(
         ImmutableMap.of(
-            "kafka-cluster", "kafka1",
-            "ksql-cluster", "ksql1"),
+            "path", Collections.emptyList(),
```

Comment:
nit: can you switch this test from using `assertEquals` to hamcrest's `assertThat` please?